### PR TITLE
[UXIT-1823] Fix Layout Shifts for FilterMenu

### PR DESCRIPTION
**Pull request by barbaraperic**
## 📝 Description

Replace flexbox with grid to prevent layout shift.

**Type:** Bug fix

https://github.com/user-attachments/assets/582a104c-fb90-46d5-9d02-637e8d63754f

### CODE REVIEW

**Review by mirhamasala — COMMENTED** [review]
Oh! This is so great! That layout shift was embarrassing. 😂 

I do have one comment! 🙏🏼

File: src/app/_components/CategorySidebar.tsx [hunk 1]
```diff
@@ -54,17 +54,7 @@ CategorySidebar.Container = function List({
...
-      className={clsx(
-        'space-y-4',
-        touchTarget.offsetClassX,
-        touchTarget.offsetClassY,
-      )}
-    >
-      {children}
-    </Component>
-  )
+  return <Component className="space-y-4">{children}</Component>
```
Comment (mirhamasala) [inline src/app/_components/CategorySidebar.tsx:57]: @barbaraperic - Why did we remove:

```
        touchTarget.offsetClassX,
        touchTarget.offsetClassY,
        
        ```

Comment (barbaraperic) [reply src/app/_components/CategorySidebar.tsx:57]: HA yes, you're right 👍

**Review by mirhamasala — APPROVED** [review]
Let's go before someone dogfoods our site! 🫥❤️

**Pull request merged**